wasm2c/wasm-rt-no-sandbox-declarations.h

```diff
@@ -106,4 +106,15 @@ DEFINE_STORE(i64_store32, u32, u64)
 
 // TODO: floating point
 
+extern u32 open(u64, u32, ...);
+
+inline static u32 wasm_open2(u64 p, u32 f) {
```

Comment:
where are they called from?

Reply:
They are in a small library file that needs to be WASM-compiled.

See https://github.com/AndroidWasm/wasm_ndk/pull/11 - specifically open-proxy.c